strings/nuclear_disk_quips.json

```diff
@@ -0,0 +1,189 @@
+		"I wonder if there's a bar on this shuttle. I don't do well with liquids, but you... might?",
+		"It feels good to be a real thing on a real person.",
+		"Nanotrasen will be pleased when they find out I've made it back safe and sound.",
+		"Remember to buckle up so you don't flop on the floor!",
+		"Stand clear of those doors, they'll suck both of us out into space.",
+		"The floor here seems a touch cleaner than what we had to deal with on the station. Floors are nice, though.",
+		"Were there any mishaps? If all goes well as we go back, all will be forgotten.",
+		"You know, I may be small, but this escape shuttle is CRAMPED."
+	],
+	"On Syndicate Shuttle":
+	[
+		"Don't forget me in that cramped-ass nuke.",
+		"DON'T HURT ME PLEASE FOR THE LOVE OF GOD DON'T HURT ME!!!",
+		"FUCK YOU SYNDICATE NUCLEAR OPERATIVES WE RIDING LOCO WITH NANOTRASEN!!!!",
+		"HAHAHA YOU LEFT A DOSSIER OF SECRET PLANS AND... oh, those are just intructions on how to bomb the station. We already knew that. Nevermind.",
+		"I resign myself to my inevitable fate.",
+		"I smell fish.",
+		"I SMELL WAFFLE CO. MERCHANDISE. OPEN A WINDOW, I'M GOING TO HURL.",
+		"I think I didn't like Nanotrasen that much? Does that help?",
+		"I'm not sure I like this place.",
+		"I'm not sure I like you.",
+		"I'm okay with just sitting here.",
+		"Listen, just put me in that slot, and I'll probably shut up. No guarantees, though.",
+		"Man, those turrets are a doozy. Good thing I don't get shot by them personally.",
+		"Oh, that diorama in the back is actually kinda cute.",
+		"That ice planet of yours fucking sucks.",
+		"That's some nice Gorlex gear you got there. Which R&D department did you steal it from?",
+		"There's... a lot of red in here.",
+		"Uhm... nice posters...",
+		"Y'know, I'm not sure I like you. I'm not sure I like you at all.",
+		"Y'think TerraGov cares about these warheads?",
+		"YOU'RE GOING TO KILL ME, AREN'T YOU?!"
+	],
+	"On Unfavorable Shuttle":
+	[
+		"Are you SURE you're in the right place?",
+		"BE FUCKING CAREFUL WITH THAT CONSOLE!",
+		"Hey, be careful about sending this shuttle.",
+		"Hey, I don't think you should send this shuttle. At least leave me behind.",
+		"I hear the weather's shit where we're going.",
+		"I looked at the destination board and it's not looking good for us.",
+		"I'm a clingy bastard who likes the station, and this isn't looking good at ALL.",
+		"I'm not pissed off now, but I will be very soon",
+		"My code is sternly rejecting spending another second on this shuttle.",
+		"Ruh-roh, this isn't going to a good spot for me.",
+		"Wait, I don't like where this shuttle is going.",
+		"WAIT, WHAT THE FUCK ARE YOU DOING ON THIS SHUTTLE?!",
+		"WOAH, don't wanna send me there, haha.",
+		"You know, maybe this destination selector is lying. Maybe you know what you're doing. But do you really?"
+	],
+	"On Station":
+	[
+		"Ah, good times in a good place.",
+		"Do you like big chungus memes?",
```

Comment:
fuck you

Reply:
i know i left this in from when i was debugging the system but i'm conflicted on removing it